python/earthquake-tracker/earthquake_tracker/main.py

```diff
@@ -14,7 +14,7 @@
 from sqlalchemy import text
 from sqlalchemy.dialects.postgresql import insert
 
-dbos = DBOS()
+DBOS()
```

Comment:
Modify the comment to say this configures DBOS. It's no longer "create a DBOS app"